tests/data-files/S2A_MSIL1C_20210908T042701_N0301_R133_T46RER_20210908T070248.SAFE/expected_output.json

```diff
@@ -100,7 +100,7 @@
     },
     {
       "rel": "self",
-      "href": "/var/folders/3q/jbg6x0zx3194zq6_2jbwygjw0000gn/T/tmpkytdhlgs/S2A_MSIL1C_20210908T042701_R133_T46RER_20210908T070248.json",
+      "href": "/var/folders/3q/jbg6x0zx3194zq6_2jbwygjw0000gn/T/tmp9ewihdpa/S2A_MSIL1C_20210908T042701_R133_T46RER_20210908T070248.json",
```

Comment:
I'd suggest removing the self link from the expected output so we don't get diff churn here every time.